examples/Makefile

```diff
@@ -0,0 +1,20 @@
+CC = oshcc
+
+.PHONY: all
+all:
+	${CC} hello.c -o hello
+	${CC} pi.c -o pi
+	${CC} pi_reduce.c -o pi_reduce
+
+hello: hello.c
+	${CC} hello.c -o $@
+
+pi: pi.c
+	${CC} pi.c -o $@
+
+pi_reduce: pi_reduce.c
+	${CC} pi_reduce.c -o $@
+
+.PHONY: clean
+clean:
+	${RM} *.o hello pi pi_reduce
```

Comment:
Should we use `${RM}` here? It may not have been defined by the user / system.

Reply:
Yes, I also see it everywhere. We can keep it as it is I think. Got a useful read: https://www.gnu.org/software/make/manual/html_node/Implicit-Variables.html